### ISSUE TITLE
New Functionality - Build Plans

### ISSUE THREAD

**Issue by maphillips1** [maphillips1/EveHelperWF#11]
Work on a build plan screen for more complicated items. 
	1.) This screen should only allow one final product. 
	2.) THe Final product can't be changed. You select this when starting a new build plan.
	3.) You should be able to select to invent, build, or react each blueprint or reaction in the chain. 
	4.) There should be one place for everything you have to buy. There should be a copy to clip board button. 
		this should put it in a format that Eve can consume on the multibuy screen. 
	5.) The screen should have a button to optimize the number of final products to build at a time for the child products. 
		- This means that if a blueprint requires 30 items of a child blueprint, but the batch size is 200, 
		 the tool should tell you the optimum number of runs for the final product is 6. This utilizes the 
		 child products in the best possible manner. 
		- Optimization routines will take a while and should be the last thing added. 
	6.) The screen will have to draw it's own production graph. FOr the most complicated blueprints like capital t2 ships
	    the build plan will have to be easy to read and not overwhelming. this is the most high end use. If I can ge that to work, 
	    everything else will.

**Comment by maphillips1** [comment]
This has mostly been completed with commit bb61981. I still need to do some testing. Correct a few issues I know exist with the formulas and usability, but the majority of the function now exists. 

things left before I close the ticket.

> Check the reactions input materials vs In-Game values. I think these are off currently. 
> Check the job cost vs In-Game values for both reactions and manufacturing to ensure they are correct. 
> Test the screen from scratch when you don't have anything. No default configs, no build plans. nothing. Make sure there isn't any issues with it. 
> Check for spelling mistakes. Make sure functionality is clear to a user who isn't familiar with the screen. 
> Maybe rename a few things.

**Comment by maphillips1** [comment]
With commit 032fadb, we added the blueprint store portion of the control. 

This allows a user to set ME, TE, and Max Runs per blueprint or reaction allowing for more granular control over inputs and costs.

**Comment by maphillips1** [comment]
with commit b41a119 I've added a new "Blueprint Store" tab to allow you to set BP and Reaction settings per BP or Reaction instead of having one setting for everything. This allows for more granular control. There is a "Set All" button if you want to set everything. 

I've also fixed the planetary materials requirement for the inputs. Previously the inputs were the values for 1 cycle, now, it shows the total inputs needed. 

I also corrected a calculation issue where the value would be slightly off on some items because of a rounding error.
